<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="QTabWidget" name="tabWidget_settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_units">
        <attribute name="title">
         <string>units</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_angleType">
               <property name="text">
                <string>angle display type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_angleType"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_distanceType">
               <property name="text">
                <string>distance display type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_distanceType"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_temperatureType">
               <property name="text">
                <string>temperature display type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_temperatureType"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_angleDigits">
               <property name="text">
                <string>angle display digits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_angleDigits"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_distanceDigits">
               <property name="text">
                <string>distance display digits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_distanceDigits"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_temperatureDigits">
               <property name="text">
                <string>temperature display digits</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_temperatureDigits"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_columnSettings">
        <attribute name="title">
         <string>column settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTabWidget" name="tabWidget_columnSettings">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_featureAttributes">
            <attribute name="title">
             <string>display feature attributes</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_allFeatureAttributes">
                 <property name="text">
                  <string>available attibutes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_allFeatureAttributes">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QToolButton" name="toolButton_addFeatureAttribute">
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../res/res.qrc">
                   <normaloff>:/Images/icons/1rightarrow.png</normaloff>:/Images/icons/1rightarrow.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_removeFeatureAttribute">
                 <property name="text">
                  <string>&lt;&lt;</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../res/res.qrc">
                   <normaloff>:/Images/icons/1leftarrow.png</normaloff>:/Images/icons/1leftarrow.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_up">
                 <property name="text">
                  <string>up</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_down">
                 <property name="text">
                  <string>down</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_displayFeatureAttribute">
                 <property name="text">
                  <string>displayed attributes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_displayedFeatureAttributes">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_TrafoParamAttributes">
            <attribute name="title">
             <string>display trafoParam attributes</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_allTrafoParamAttributes">
                 <property name="text">
                  <string>available attributes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_allTrafoParamAttributes">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QToolButton" name="toolButton_addTrafoParamAttribute">
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../res/res.qrc">
                   <normaloff>:/Images/icons/1rightarrow.png</normaloff>:/Images/icons/1rightarrow.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_removeTrafoParamAttribute">
                 <property name="text">
                  <string>&lt;&lt;</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../res/res.qrc">
                   <normaloff>:/Images/icons/1leftarrow.png</normaloff>:/Images/icons/1leftarrow.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="label_displayedTrafoParamAttributes">
                 <property name="text">
                  <string>displayed attributes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="listView_displayedTrafoParamAttributes">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_plugins">
        <attribute name="title">
         <string>plugins</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTreeView" name="treeView_plugins">
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>0</height>
                </size>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="autoExpandDelay">
                <number>-1</number>
               </property>
               <property name="animated">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                   <strikeout>false</strikeout>
                  </font>
                 </property>
                 <property name="text">
                  <string>plugin description:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="txt_pluginDescription">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>70</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>author:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txt_pluginAuthor">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>version:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txt_pluginVersion">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>file path:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="txt_pluginFilePath">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="widget_pluginInfo" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_others">
        <attribute name="title">
         <string>others</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_differences">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>show differences</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
